plugins/out_retry/retry.c

```diff
@@ -98,11 +90,24 @@ static int cb_retry_exit(void *data, struct flb_config *config)
     return 0;
 }
 
+/* Configuration properties map */
+static struct flb_config_map config_map[] = {
+    {
+        FLB_CONFIG_MAP_STR, "retries", NULL,
```

Comment:
the old code specifies a default value, if you don't set one, you are breaking backward compatibility.